swagger/docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: '3.3'
+
+services:
+
+  gateway:
+    image: nginx:alpine
+    ports:
+      - 8888:80
+    volumes:
+      - ./nginx:/etc/nginx:ro
+    links:
+      - account:account.sumelms.local
+
+  # Fake APIs
+
+  account:
+    image: muonsoft/openapi-mock
+    environment:
+      OPENAPI_MOCK_SPECIFICATION_URL: 'https://raw.githubusercontent.com/sumelms/microservice-account/main/swagger/account-api.yaml'
```

Comment:
This URL is gonna be valid only after [this PR](https://github.com/sumelms/microservice-account/pull/14) gets merged.